OrbitLinuxTracing/TracerThread.h

```diff
@@ -157,6 +158,9 @@ class TracerThread {
   std::unique_ptr<PerfEventProcessor> uprobes_event_processor_;
   std::unique_ptr<GpuTracepointEventProcessor> gpu_event_processor_;
 
+  TracepointInfoToHashSet instrumented_tracepoints_ids_;
+  absl::flat_hash_map<uint64_t, orbit_grpc_protos::TracepointInfo> stream_id_to_tracepoint_;
```

Comment:
Move these up right after `callchain_sampling_ids_`, so that all members related to stream ids are together.

Reply:
Of course. Thank you!